### PR TITLE
add fact gathering directly after installimage to avoid uncessary cus…

### PR DESCRIPTION
**Pull request by jeichler**
…tom checks

### CODE REVIEW

File: tasks/installimage.yml [hunk 1]
```diff
@@ -63,36 +63,35 @@
...
-  raw: 'echo -n $(test -f /etc/redhat-release && echo "RedHat" && exit 0 || test -f /etc/debian_version && echo "Debian" && exit 0 || test -f /etc/SuSE-release && echo "Suse" && exit 0 || echo "Unknown")'
-  register: os_switch
-  changed_when: false
+- name: Clear gathered facts from all currently targeted hosts
+  meta: clear_facts
+
+- name: update facts since we now have our OS installed
+  setup:
 
 # uses raw instead of module as python might ont be installed yet
```
Comment (geektoor) [inline tasks/installimage.yml]: not

File: tasks/installimage.yml [hunk 2]
```diff
@@ -53,7 +53,7 @@
     host: "{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}"
     port: 22
     delay: 1
-    timeout: 180
+    timeout: "{{ hetzner_installimage_wait_timeout_after_reset }}"
   delegate_to: localhost
 
 - name: add ssh keys per host
```
Comment (geektoor) [inline tasks/installimage.yml]: as it might be puzzling, we should call the task "ssh host keys" or something

**Review by geektoor — COMMENTED** [review]
Will test it tomorrow, just two small notes if you want to fix them (even you didn't introduce them ;)

File: tasks/installimage.yml (hunk 1 above)
Comment (mophahr) [inline tasks/installimage.yml]:
```suggestion
# uses raw instead of module as python might not be installed yet
```

**Review by mophahr — COMMENTED** [review]
minor comments aside, this looks fine

**Pull request merged**